release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -63,11 +63,31 @@ For each release, the schedule with deliverables is added to the release directo
     - [Clean up access](#clean-up-access)
 - [Celebrate](#celebrate)
 
+---
+
+## Overview
+
+For each release, the schedule with deliverables is added to the release directory. This section talks about specific Docs Lead deliverables for each milestone in the release timeline.
+
+Read the release timeline and **make sure the timeline includes deadlines for documentation work**, e.g: [1.21 timeline](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.21#timeline):
+
+- Docs deadline - Open placeholder PRs (~3.5 weeks before release and shortly after Code Freeze)
```

Comment:
We should explicitly state who is responsible for opening the placeholder PRs. In my experience it's the KEP owners or someone from the same SIG - as opposed to members of the Docs release team.

Reply:
Put the reminders is responsible of the Docs team, I will try to improve the text